### ISSUE TITLE
Sugestões para _Dashboard_

### ISSUE THREAD

**Issue by joao-nunes** [dssg-pt/mp-violencia-domestica#2]
Visualização:

- Heatmaps:

1. Tendência (Crescimento/Manutenção/Redução)
2. Incidência violência doméstica, por níveis. (e.g. muito baixa, baixa, moderada, elevada, muito elevada)
3. Desvio em relação ao distrito, por níveis (e.g. muito inferior, inferior, igual, superior, muito superior)
4. Desvio em relação ao país, por níveis (e.g. muito inferior, inferior, igual, superior, muito superior)

- Plots:

1. Evolução temporal da Taxa de Desemprego, Bruta Nupcialidade e Bruta Divorcialidade comparada com a evolução temporal do  número de casos de Violência Doméstica
2. Evolução temporal (por ano) do número de casos de violência doméstica por concelho (dados disponíveis desde 2001)
3. Gráfico Cumulativo do número anual de casos registados (total e por concelho)

Hipóteses a testar (e.g. Apresentar resultados numa matriz de correlação):
Correlação entre o número de casos de Violência Doméstica e:

- Taxa Bruta Nupcialidade 
- Taxa Bruta de Divorcialidade
- Taxa de Desemprego 
- Nível de escolaridade 
-  Poder de compra
- Magnitude Salarial

**Comment by RuiMedon** [comment]
Nice! Está aqui um bom ponto de partida!
Alguns comentários para pensarmos quando fizermos um sync:
- Devemos ligar as fontes às métricas
- Quais os thresholds para as categorias baixa, moderada, elevada, muito elevada? Porquê uma "muito elevada" e não uma "muito baixa"?
- Como se calcula a incidência?
- Não percebi o que é a "distância em relação ao distrito/país"
- Devemos testar primeiro a correlação entre os casos e taxa de semprego / nupcialidade / divorcialidade antes de colocar num gráfico
- Evolução de casos por ano, sem dúvida. Adicionando o concelho fica uma visualização muito saturada.
- Gosto da ideia de testar as correlções entre os fatores que referes no fim. Não gosto tanto da ideia de mostara a matriz de correlação. Um user menos versado em estatística não iria compreender o que significa

**Comment by RuiMedon** [comment]
A minha proposta seria a seguinte:
Métrica principal: 
- Nº casos até agora + Nº médio de casos por dia (queria que passasse uma impressão mais grave, mas receio que seja um pouco alarmista)

Secção 1 - Perfil da vítima:
- Género (gráfico de barras)
- Idade (histograma)
- Estado civil (gráfico de barras)
- Distrito (heatmap)
- Estado civil (gráfico de barras)
- Relação com o autor do crime (gráfico de barras)

Secção 2 - Histórico:
- Timeseries de nº de casos

**Comment by joao-nunes** [comment]
Rui, as tuas observações foram muito pertinentes. Vou já alterar algumas coisas para as minhas ideias ficarem mais claras. A incidência seria o número de casos por cada 100 000 habitantes, por unidade de tempo (ano).  No entanto, falamos melhor na reunião. Será mais fácil explicar o meu raciocínio.

**Comment by RuiMedon** [comment]
Sim, falamos lá. Acho que agora temos que discutir estas coisas mais ativamente.
Estava a pensar se aqueles números relacionados com as "não respostas" seriam interessantes de colocar aqui. A ideia seria passar uma noção do receio das pessoas de falar no assunto. Food for thought...

**Comment by RuiMedon** [comment]
![WhatsApp Image 2021-03-06 at 11 39 10](https://user-images.githubusercontent.com/49407522/110205621-ef3b2600-7e70-11eb-8f49-fe72e7887526.jpeg)

**Comment by RuiMedon** [comment]
- Overview (atual) (estático)
	- Quantos casos ocorrem (em média) por ano?
		- Numérico
	- A situação está a melhorar ou piorar?
		- Percentagem
	- Quem são as vítimas de violência doméstica?
		- Sexo
			- Top 1: Categórico + percentagem
		- Idade
			- Top 1: Categórico + percentagem
		- Relação c/ autor
			- Top 1: Categórico + percentagem
	- Quem são os autores de violência doméstica?
		- Sexo
			- Top 1: Categórico + percentagem
		- Idade
			- Top 1: Categórico + percentagem
		- Relação c/ vítima
			- Top 1: Categórico + percentagem
	- Quais as zonas mais afetadas?
		- Heatmap geográfico
	- Qual o motivo da violência doméstica? (não sei se temos)
		- Top 1: Categórico + percentagem
	- Qual o tipo de violência doméstica?
		- Top 1: Categórico + percentagem

**Comment by RuiMedon** [comment]
- Drill down (dinâmico)
	- Selector de dimensão
	- Filtros
		- Ano
		- Sexo
		- Idade
		- Relação c/ agressor
		- (...)
	- Gráficos
		- Evolução temporal
		- Sexo
		- Idade
		- Relação c/ agressor
		- (...)

**Comment by RuiMedon** [comment]
![image](https://user-images.githubusercontent.com/49407522/113427602-3ca5a700-93cd-11eb-98ec-c63b0f50ebdc.png)

**Comment by RuiMedon** [comment]
![image](https://user-images.githubusercontent.com/49407522/113427619-46c7a580-93cd-11eb-8d62-60d6b7571b3f.png)
